r/sedonadb/README.md

```diff
@@ -30,6 +30,13 @@ SQL with high function coverage and GeoParquet IO.
 
 ## Installation
 
+sedonadb can be installed from
+[R-multiverse](https://community.r-multiverse.org/):
+
+``` r
+install.packages("sedonadb", repos = "https://community.r-multiverse.org")
+```
```

Comment:
I noticed that R-multiverse doesn't have wk, so I just submitted a PR to add it, and I think it will be available within the next few hours.
r-multiverse/contributions#256